modules/functional-test/pom.xml

```diff
@@ -142,5 +142,10 @@
             <artifactId>retrofit1-okhttp3-client</artifactId>
             <version>1.0.2</version>
         </dependency>
+		<dependency>
+		    <groupId>mysql</groupId>
+		    <artifactId>mysql-connector-java</artifactId>
+		    <version>5.1.6</version>
+		</dependency>

```

Comment:
I just annotate it here, but it applies to multiple files: In general we indent with four spaces. One of the files below has 8, one has 2. This one has four, but the dependency should by one level closer to the root.